website/docs/r/backup_policy_vm.html.markdown

```diff
@@ -166,6 +168,14 @@ The `retention_yearly` block supports:
 
 ---
 
+A `tiering_policy` block supports the following:
+
+* `tiering_mode` - (Required) The tiering mode to control automatic tiering of recovery points. Possible values are `TierAfter` and `TierRecommended`.
```

Comment:
i think this can just be mode as tiering is is in the block name?

Reply:
Updated

Reply:
Renamed it to "mode".